src/core/hle/service/nvdrv/nvdrv.h

```diff
@@ -1,16 +1,21 @@
-// SPDX-FileCopyrightText: Copyright 2018 yuzu Emulator Project
-// SPDX-License-Identifier: GPL-2.0-or-later
+// SPDX-FileCopyrightText: 2021 yuzu emulator team and Skyline Team and Contributors
+// (https://github.com/skyline-emu/)
+// SPDX-License-Identifier: GPL-3.0-or-later Licensed under GPLv3
+// or any later version Refer to the license.txt file included.
```

Comment:
By the way, why do the copyright notices mention the year 2021 and not 2022?

```suggestion
// SPDX-FileCopyrightText: 2021 yuzu Emulator Project
// SPDX-FileCopyrightText: 2021 Skyline Team and Contributors <https://github.com/skyline-emu>
// SPDX-License-Identifier: GPL-3.0-or-later
```